script.js

```diff
@@ -0,0 +1,168 @@
+    img_source: 'images/Snapshoot_Portfolio.svg',
+    button_others: ['See Live', 'See Source'],
+    position: 'left',
+    sbutton: 'See Project'
+  },
+
+  {
+    id: 2,
+    title: 'Multi-Post Stories',
+    subtitile: 'FACEBOOK',
+    mid_list : ['Full Stack Dev', '2015'],
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    list_stack: ['html', 'rails', 'css', 'javascript'],
+    img_source: 'images/Snapshoot_PortfolioB.svg',
+    button_others: ['See Live', 'See Source'],
+    position: 'right',
+    sbutton: 'See Project'
+  },
+  {
+    id: 3,
+    title: 'Facebook 360',
+    subtitile: 'FACEBOOK',
+    mid_list : ['Full Stack Dev', '2018'],
+    description: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    list_stack: ['html', 'css', 'javascript'],
+    img_source: 'images/Snapshoot_PortfolioC.svg',
+    button_others: ['See Live', 'See Source'],
+    position: 'left',
+    sbutton: 'See Project'
+  },
+  {
+    id: 4,
+    title: 'Uber Navigation',
+    subtitile: 'Uber',
+    mid_list : ['Lead Developer', '2015'],
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    list_stack: ['html', 'css', 'javascript'],
+    img_source: 'images/Snapshoot_PortfolioD.svg',
+    button_others: ['See Live', 'See Source'],
+    position: 'right',
+    sbutton: 'See Project'
+  },
+];
+
+
+/************************** HAMBURGER SECTION **************************************/
+const menuItem = document.querySelector('.mobile-links');
+const closeMobileItems = document.querySelectorAll('.closeMenu');
+const displayMobileMenu = () => {
+  menuItem.style.display = 'block';
+};
+const closeMobileMenu = () => {
+  menuItem.style.display = 'none';
+};
+document.querySelector('.openMenu').addEventListener('click', displayMobileMenu);
+closeMobileItems.forEach(((element) => element.addEventListener('click', closeMobileMenu)));
+
+
+
+//Build HTML Element
+function buildElement(img_source, title, subtitle, listA, listB, description, StackArray, id, btnText) {
+  document.querySelector('#maingrid').innerHTML += `
+  <div class="flex-container" id="data-${id}">
+  <div class="flex-items" ><img src="${img_source}" alt="Portfolio" /></div>
+  <div class="flex-items">
+    <h2>${title}</h2>
+    <span class="subtitle">${subtitle} </span>
+    <span class="mid-list">
+      <span class="dot"></span>${listA}
+      <span class="dot"></span>${listB}
+    </span>
+ 
+    <p class="description">${description}</p>
+    <ul class="list-stack">
+    ${StackArray.join(' ')}
+    </ul>
+    <button type="button" class="show_modal"  data-id="${id}">${btnText}</button>
+  </div>
+ </div>`
+}
+
+// Looping the sections
+
+project_array.forEach((projectEl, index, array) => {
```

Comment:
- [ ] Your modal window is looking great! Let's make it even better by moving the buttons (see live, see source) to the right as the template shows:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-05-11 a la(s) 6 51 13 p  m](https://user-images.githubusercontent.com/83717285/167966328-aec9469c-211a-4a66-8cc3-34ac553ed5a3.png)|![Captura de pantalla 2022-05-11 a la(s) 6 50 51 p  m](https://user-images.githubusercontent.com/83717285/167966358-f866e78b-29be-4c34-bf97-6d2d8cdec3d6.png)|